aiida_mlip/calculations/base.py

```diff
@@ -44,6 +46,17 @@ def validate_inputs(
             raise InputValidationError(
                 "Structure must be specified through 'struct' or 'config'"
             )
+    if (
+        "arch" not in inputs
+        and "model" not in inputs
+        and (
+            "config" not in inputs
+            or ("arch" not in inputs["config"] and "model" not in inputs["config"])
+        )
+    ):
+        raise InputValidationError(
+            "Either 'arch' or 'model' must be specified in the inputs or config"
```

Comment:
```suggestion
            "'arch' and 'model' must both be specified in the inputs or config"
```
Otherwise it reads as one or the other, to me

Reply:
I did mean what I wrote but we do actually want what you wrote (which might mean also I need to change something in the rest of the code)

Reply:
Sorry you're right, I didn't read the logic carefully enough. I guess it depends how/where you treat defaults etc.